MVP Module/Source/Feed UI/Controllers/FeedViewController.swift

```diff
@@ -8,7 +8,9 @@ protocol FeedViewControllerDelegate {
 	func didRequestFeedRefresh()
 }
 
-public final class FeedViewController: UITableViewController, UITableViewDataSourcePrefetching, FeedLoadingView {
+public final class FeedViewController: UITableViewController, UITableViewDataSourcePrefetching, FeedLoadingView, FeedErrorView {
+	@IBOutlet weak var errorView: ErrorView!
```

Comment:
`errorView` doesn't need to be weak. Also, the setter can be private to prevent clients from changing the instance.

```suggestion
	@IBOutlet private(set) var errorView: ErrorView!
```